tests/Auth0.ManagementApi.IntegrationTests/OrganizationTests.cs

```diff
@@ -189,6 +192,40 @@ public async Task Test_organization_connections_crud_sequence()
             }
         }
 
+        [Fact]
+        public async Task Test_organization_connections_show_as_button_crud_sequence()
+        {
+            var createConnectionResponse = await fixture.ApiClient.Organizations.CreateConnectionAsync(ExistingOrganizationId, new OrganizationConnectionCreateRequest
+            {
+                ConnectionId = ExistingConnectionId,
+                ShowAsButton = false
+            });
+
+            createConnectionResponse.Should().NotBeNull();
+            createConnectionResponse.ShowAsButton.Should().Be(false);
+
+            try
+            {
+                var updateConnectionResponse = await fixture.ApiClient.Organizations.UpdateConnectionAsync(ExistingOrganizationId, ExistingConnectionId, new OrganizationConnectionUpdateRequest
+                {
+                    ShowAsButton = true
+                });
+
+                updateConnectionResponse.Should().NotBeNull();
+                updateConnectionResponse.ShowAsButton.Should().Be(true);
+
+                var connection = await fixture.ApiClient.Organizations.GetConnectionAsync(ExistingOrganizationId, ExistingConnectionId);
+
+                connection.Should().NotBeNull();
+                connection.ShowAsButton.Should().Be(true);
+            }
+            finally
+            {
+                // Unlink Connection
+                await fixture.ApiClient.Organizations.DeleteConnectionAsync(ExistingOrganizationId, ExistingConnectionId);
+            }
+        }
```

Comment:
I think we can avoid this test, and instead use the original to:

- Leave the default on create and assert its true
- Set it to false on update and assert its false after

Reply:
Done